src/components/DecksPickBreakdown.js

```diff
@@ -74,7 +74,17 @@ const DecksPickBreakdown = ({ draft, seatIndex, deck, defaultIndex }) => {
 
 DecksPickBreakdown.propTypes = {
   draft: PropTypes.shape({
```

Comment:
we should use a DraftPropType for this validation

Reply:
Done in #1798